app/features/analysis/cards/OneRM/OneRMView.js

```diff
@@ -68,19 +72,75 @@ class OneRM extends Component {
         this.props.changeDays(value);
     }
 
+    _displayTagsToInclude() {
+        if (this.props.tagsToInclude === undefined || this.props.tagsToInclude === null || this.props.tagsToInclude.length === 0) {
+            return (<Text style={[styles.tagText, styles.placeholderText]}>Tags to Include</Text>);
+        }
+
+        let position = 0;
+
+        const pills = this.props.tagsToInclude.map((tag) => {
+            return <Pill key={position++} text={tag} style={{paddingRight: 5, paddingBottom: 3, paddingTop: 3}} />
+        });
+
+        return (<View style={styles.tagField}>{pills}</View>);
+    }
+
+    _renderTagsToInclude() {
+        return (
+            <View style={[styles.field, {flex: 1}]}>
+                <TouchableHighlight onPress={() => this.props.tappedTagsToInclude()} underlayColor='#e0e0e0'>
+                    {this._displayTagsToInclude()}
+                </TouchableHighlight>
+            </View>
+        );
+    }
+
+    _displayTagsToExclude() {
+        if (this.props.tagsToExclude === undefined || this.props.tagsToExclude === null || this.props.tagsToExclude.length === 0) {
+            return (<Text style={[styles.tagText, styles.placeholderText]}>Tags to Exclude</Text>);
+        }
+
+        let position = 0;
+
+        const pills = this.props.tagsToExclude.map((tag) => {
+            return <Pill key={position++} text={tag} style={{paddingRight: 5, paddingBottom: 3, paddingTop: 3}} />
+        });
+
+        return (<View style={styles.tagField}>{pills}</View>);
+    }
+
+    _renderTagsToExclude() {
+        return (
+            <View style={[styles.field, {flex: 1}]}>
+                <TouchableHighlight onPress={() => this.props.tappedTagsToExclude()} underlayColor='#e0e0e0'>
+                    {this._displayTagsToExclude()}
+                </TouchableHighlight>
+            </View>
+        );
+    }
+
     render() {
         if (Platform.OS === 'ios') {
```

Comment:
I tried to resolve the merge conflicts best I could to just make this work, but not sure if it's correct

- [ ] Make tags show up on android now
- [ ] Confirm it's still fine on iOS